tensorflow/compiler/tests/einsum_op_test.py

```diff
@@ -61,13 +60,6 @@ def testMatMul(self):
           np.array([[8]], dtype=dtype),
           expected=np.array([[-2]], dtype=dtype))
 
-      with compat.forward_compatibility_horizon(2019, 10, 19):
```

Comment:
You probably should just delete this line and leave the `self._testBinary` there?

Reply:
The git diff here is a bit misleading. The same test is executed directly above. Just removing the `compat` line would execute the same test twice which would be unnecessary. The same is true for the other changes in this file.